src/applications/personalization/profile/components/ProfileHeader.jsx

```diff
@@ -124,6 +138,26 @@ const ProfileHeader = ({
                 {getServiceBranchDisplayName(latestBranchOfService)}
               </dd>
             )}
+            {showUpdatedHeader &&
+              totalDisabilityRating && (
+                <>
+                  <dt className="sr-only">total disability rating</dt>
```

Comment:
I know I asked for this but I do wonder how this will work for a screen reader user. It will read this hidden `dt`. But then the `dd` text is in an `a` tag with an `aria-label`. So will the screen reader say: "total disability rating. view your disability rating." 

In other words, if an `a` tag has an `aria-label`, will a screen reader speak both the `aria-label` _and_ the `a` tag's contents? Or just the `aria-label`?

Based on [this post](https://www.deque.com/blog/text-links-practices-screen-readers/), it seems like it depends on the screen reader :( @joshkimux do you have any thoughts on how to best handle this? This is not something that will be in prod any time soon so we'll likely merge before you can respond but we can loop back later. The design and copy for this might change anyway.

Reply:
Good question, would also love to hear back from Josh about this! 💯

Reply:
> In other words, if an a tag has an aria-label, will a screen reader speak both the aria-label and the a tag's contents? Or just the aria-label?

It's supposed to just read the aria-label, so it would be "total disability rating. view your disability rating."

Reply:
Maybe if the `aria-label` was something like `view details about your 70% total disability rating`? ¯\_(ツ)_/¯

Reply:
@erikphansen Screen readers should prioritize the `aria-label` before the `title`, and only the `aria-label` should be read. In other words, `aria-label` should do the trick. In general, I'd caution to [avoid using/relying on the `title` attribute as documented in this article](https://www.24a11y.com/2017/the-trials-and-tribulations-of-the-title-attribute/) ✌️ 

I'm bad at reading code, but I'll be glad to test this on screen readers once it's ready 😄